.moai/specs/SPEC-ASTGREP-DOGFOOD-CLEANUP-001/spec.md

```
@@ -136,22 +152,28 @@ ast-grep pre-tool gate는 기본 OFF (`AstGrepGate.Enabled` 컴파일 기본값
   (2) 해당 디렉터리가 오늘 활발히 수정 중(untracked `credentials.yml` in-flight)이라 병렬 세션
   레이스 위험이 있음. 영어화는 이 룰들이 템플릿으로 승격될 때 배포 SPEC이 처리한다.
 
-### Out of Scope — ast-grep gate 활성화 및 Go 코드
-- `AstGrepGate.Enabled` 활성화, gate 로더 추가, `moai ast-grep` CLI 동작 변경 등 Go 코드
-  (`internal/`/`pkg/`/`cmd/`) 변경은 본 SPEC 범위 밖이다. 본 SPEC은 순수 파일/설정 위생 작업이다.
+### Out of Scope — ast-grep gate blocking 활성화 및 Go 코드
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)